src/hotspot/share/opto/superword.cpp

```diff
@@ -1996,291 +1961,20 @@ void PackSet::verify() const {
 }
 #endif
 
-// The PacksetGraph combines the dependency graph with the packset. In the PackSet
-// graph, we have two kinds of nodes:
-//  (1) pack-node:   Represents all nodes of some pack p in a single node, which
-//                   shall later become a vector node.
-//  (2) scalar-node: Represents a node that is not in any pack.
-// For any edge (n1, n2) in the dependency graph, we add an edge to the PacksetGraph for
-// the PacksetGraph nodes corresponding to n1 and n2.
-// We work from the dependency graph, because it gives us all the data-dependencies,
-// as well as more refined memory-dependencies than the C2 graph. The dependency graph
-// does not have cycles. But packing nodes can introduce cyclic dependencies. Example:
-//
-//                                                       +--------+
-//  A -> X                                               |        v
-//                     Pack [A,B] and [X,Y]             [A,B]    [X,Y]
-//  Y -> B                                                 ^        |
-//                                                         +--------+
-//
-class PacksetGraph {
```

Comment:
Note: graph refactored into `VTransformGraph`. Comments above moved to `VTransformGraph::schedule`.